src/main/java/frc/team1816/robot/Robot.java

```diff
@@ -31,7 +36,9 @@ public void disabledInit() { }
     public void autonomousInit() { }
 
     @Override
-    public void teleopInit() { }
+    public void teleopInit() {
+        Components.getInstance().drivetrain.setDefaultCommand(new GamepadDriveCommand());
```

Comment:
not a show stopper, but drivetrain should be private.  we should either be using getters/setters in Components or using Drivetrain.getInstance()

Reply:
If I recall, we ran into issues with that Singleton styling when trying to implement in for Zenith, but I'm not 100% sure. Will have to test it.